customer-commons/src/main/java/no/unit/nva/customer/model/CustomerMapper.java

```diff
@@ -17,6 +24,22 @@
     private CustomerMapper() {
     }
 
+    private static CustomerDtoWithoutContext toCustomerDtoWithoutContext(CustomerDto customerDto) {
+        return objectMapper.convertValue(customerDto, CustomerDtoWithoutContext.class);
+    }
+
+    public static CustomerList toCustomerListFromCustomerDtosWithoutContexts(
+            List<CustomerDtoWithoutContext> customerDtos) {
+        URI id = URI.create(getIdNamespace());
+        return new CustomerList(id, customerDtos);
```

Comment:
id should not be passed as a parameter. NAMESPACE is a public static variable of the system.